libpod/image/parts.go

```diff
@@ -16,6 +17,11 @@ type imageParts struct {
 	hasRegistry bool
 }
 
+// Registries must contain a ":" or a "." or be localhost
+func isRegistry(name string) bool {
+	return strings.ContainsAny(name, ".:") || name == "localhost"
+}
```

Comment:
Could a repo contain a period?

Reply:
Yes, and in that case the user must specify `example.com/repo.with.dots`. This is consistent throughout the docker/distribution ecosystem.

_Please_ don’t invent new syntaxes.

Reply:
> must specify

… the host name explicitly, e.g. …